conjure-java-undertow-runtime/src/test/java/com/palantir/conjure/java/undertow/runtime/ConjureExceptionHandlerTest.java

```diff
@@ -138,13 +138,15 @@ static Stream<Arguments> differentPackageExceptions() {
                 Arguments.of((Supplier<Exception>) () -> new DifferentPackageEndpointException(null)));
     }
 
+    @SuppressWarnings("removal")
```

Comment:
This should be removed when we remove the `CheckedServiceException`. The constructors of `CheckedServiceException` have been deprecated.